readme.md

```diff
@@ -383,21 +436,22 @@ juan.full_name
 
 `attr_accessor` creates getters and setters.
 
-## Break  (10:50 - 11:00, 10 min)
+-------
+
+## Break (10 minutes / 1:00)
 
-___
+-------
 
-## You do: Monkies! (11:00 - 11:20, 20 min)
+## You Do: Monkies! (20 minutes / 1:20)
```

Comment:
people always struggle w/ this one. Recommen reading through first / be prepared to talk about rspec / give hints / check in about 2 minutes through